### PR TITLE
Postgresql collations

### LINKED ISSUE #4651: [Concept Entry] Postgresql collation-expressions
### Reason/inspiration (optional)

We would like a new entry on the `Collation Expressions` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/collation-expressions/collation-expressions.md`.
 
 The entry should include:
 
 - An introductory paragraph that describes the concept
 - A ‘Syntax’ section that includes the syntax for the concept (if applicable)
 - An ‘Example’ section that shows the concept in use
 
 Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Johnnie-kay**
### Description
Created a new concept entry for Postgresql collations.

### Issue Solved
Closes #4651 [Concept Entry] Postgresql collations

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by Johnnie-kay** [comment]
Hello @cigar-galaxy82, please review this PR so i can make corrections on it to close this issue and get assigned on another one.

**Comment by cigar-galaxy82** [comment]
Sure @Johnnie-kay I will complete this by today

File: content/postgresql/concepts/collation-expressions/collation-expressions.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Below is a table of commonly used options:
+
+| Option | Syntax | Description |
+| ---- | ---- | ---- |
+| Case sensitive | `_CS` | Uppercase and lowercase letters are treated differently. "Castle" would come before "castle" during sorting. |
+| Case insensitive | `_CI` | Lower case and upper case are treated as identical, "Ban" and "ban" will be considered equal and might appear in order of other sorting rules. |
+| Accent sensitive | `_AS` |  used to consider accented and non accented letter separately. for example ‘a’ and ’ ấ’are not treated identical. |
+| Accent insensitive | `_AI` | used to consider accented and non accented letter identical. for example ‘a’ and ’ ấ’are treated identical. |
+| Width sensitive | `_WS` | Differentiates full width and half width characters present, if `_WS` wasn't mentioned then it is width-insensitive and hence full and half width characters will be considered as identical. |
```
[the remark this answers is not in the document]
Comment (cigar-galaxy82) [reply content/postgresql/concepts/collation-expressions/collation-expressions.md:112]: Use ### Example when showing code for each type collation

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]
Hi @Johnnie-kay 

I've suggested a few changes.  Also, make sure the Metadata is written properly

**Comment by cigar-galaxy82** [comment]
Hi @Johnnie-kay 

Thank you for contributing to Codecademy Docs!

I've suggested a few changes please modify. Also, make sure Metadata is  written correctly

Thank you!

**Comment by Johnnie-kay** [comment]
Hello @cigar-galaxy82, good morninng. I have made all the marked out corrections and adjustments on this issue.

**Comment by Johnnie-kay** [comment]
Hello @letsbuildanything, the asTimeZone entry file has been deleted.

**Comment by Johnnie-kay** [comment]
Hello @letsbuildanything please review this PR.

**Comment by Johnnie-kay** [comment]
Hello @cigar-galaxy82 is there anything else to do on this issue. please review and close this PR.

**Review by cigar-galaxy82 — APPROVED** [review]

**Comment by cigar-galaxy82** [comment]
hi @Johnnie-kay review from my side is done soon someone will do the second review, and then we will merge it

**Review by Sriparno08 — APPROVED** [review]
Looks good, @Johnnie-kay! 👍

**Pull request merged**